src/main/java/com/rtb/projectmanagementtool/auth/LoginDispatchServlet.java

```diff
@@ -0,0 +1,43 @@
+// Servlet called after a user logs in with Google. Tt checks to see if
+// their authId matches any entities in datastore, in which case it
+// would direct the user to the home page. If not, it directs them to
+// the page to create a new user.
+// Might need a better name than Login'Dispatch'Servlet
```

Comment:
I thought what you have here also would have been a good fit in loginServlet.  It could for example:
1.  Check if user is already logged in, send to home
2.  Check if user has auth, if not send to Google auth and redirect back to loginServlet
3.  If user has auth, find userID and redirect to home
4.  If user has auth but no userID (or if user explicitly clicks on Create User), send to create user


Both servlets as it stand now do similar actions:  check auth situation and redirect somewhere.  Keeping in 1 place I think would make it easier to deal with